### PR TITLE
[DOCS] Temporarily revert `update_expectation_suite` call in GX Cloud quickstart

### PR DESCRIPTION
**Pull request by cdkini**
Changes proposed in this pull request:
- Fix that enables this method is not yet a part of our latest release
- This change will be reverted once the release is out

### Definition of Done
- [x] My code follows the Great Expectations [style guide](https://docs.greatexpectations.io/docs/contributing/style_guides/code_style)
- [x] I have performed a [self-review](https://docs.greatexpectations.io/docs/contributing/contributing_checklist) of my own code
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

**Comment by ghost** [comment]
<details open>
<summary><b>👇 <a href="https://app.codesee.io/r/reviews?pr=7736&src=https%3A%2F%2Fgithub.com%2Fgreat-expectations%2Fgreat_expectations">Click on the image</a> for a new way to code review</b>
</summary>

####

[Review these changes using an interactive CodeSee Map](https://app.codesee.io/r/reviews?pr=7736&src=https%3A%2F%2Fgithub.com%2Fgreat-expectations%2Fgreat_expectations)

#### Legend
CodeSee Map legend
</details>

File: docs/docusaurus/docs/gx_cloud/tutorials/getting_started/getting_started_with_gx_cloud.md [hunk 1]
```diff
@@ -163,7 +163,7 @@ expectation_suite.add_expectation(
 print(expectation_suite)
 
 # Save the Expectation Suite
-context.update_expectation_suite(expectation_suite=expectation_suite)
+context.save_expectation_suite(expectation_suite=expectation_suite)
```
Comment (joshua-stauffer) [inline docs/docusaurus/docs/gx_cloud/tutorials/getting_started/getting_started_with_gx_cloud.md:166]:
```suggestion
context.save_expectation_suite(expectation_suite=expectation_suite, overwrite_existing=True)
```

**Review by joshua-stauffer — APPROVED** [review]
👍 just need to ensure we overwrite in order to update

File: docs/docusaurus/docs/gx_cloud/tutorials/getting_started/getting_started_with_gx_cloud.md (hunk 1 above)
Comment (joshua-stauffer) [reply docs/docusaurus/docs/gx_cloud/tutorials/getting_started/getting_started_with_gx_cloud.md:166]: unneccesary, already default behavior

**Pull request merged**